src/api-types.ts

```diff
@@ -59,56 +59,56 @@ export interface BlockBase {
 
 export interface ParagraphBlock extends BlockBase {
   type: 'paragraph';
-  text: RichText[];
+  paragraph: { text: RichText[] };
   children?: BlockBase[];
 }
 
 export interface HeadingOneBlock extends BlockBase {
   type: 'heading_1';
-  text: RichText[];
+  heading_1: { text: RichText[] };
   has_children: false;
 }
 
 export interface HeadingTwoBlock extends BlockBase {
   type: 'heading_2';
-  text: RichText[];
+  heading_2: { text: RichText[] };
   has_children: false;
 }
 
 export interface HeadingThreeBlock extends BlockBase {
   type: 'heading_3';
-  text: RichText[];
+  heading_2: { text: RichText[] };
```

Comment:
Hi👋, over here the key should be heading_3

Reply:
thx, fixed it